BusyTiming/NewActivitiPage/NewActivityViewController.swift

```diff
@@ -6,18 +6,60 @@
 //
 
 import UIKit
+import Combine
+
 
 class NewActivityViewController: UIViewController {
+    
+    
     private let viewModel = NewActivityViewModel()
+    private let timerVM = TimerVM()
+    private var cancellableBag = Set<AnyCancellable>()
     
     @IBOutlet weak var activityTimeLabel: UILabel!
     @IBOutlet weak var activityNameTextField: UITextField!
     @IBOutlet weak var saveButton: UIButton!
     
+   
     
-    @IBAction func cancelButtonPressed(_ sender: UIButton) {
-        dismiss(animated: true, completion: nil)
+    override func viewDidLoad() {
+         super.viewDidLoad()
+        
+        viewModel.activityTimePublisher
+        .assign(to:\.text, on: activityTimeLabel)
+            .store(in: &cancellableBag)
+        
+        viewModel.saveIsEnablePubliser
+            .assign(to: \.isEnabled, on: saveButton)
+            .store(in: &cancellableBag)
+        
+        viewModel.$newActivityName.map{$0 as String}.assign(to:\.text, on: activityNameTextField)
+            .store(in: &cancellableBag)
+        
+        activityNameTextField.textPublisher().assign(to: \.newActivityName, on: viewModel)
+            .store(in: &cancellableBag)
     }
-    @IBAction func saveButtonPressed(_ sender: UIButton) {
+     
+     override func viewWillAppear(_ animated: Bool) {
+         super.viewWillAppear(animated)
+         viewModel.getTimeActivity()
```

Comment:
have you tried to move the `getTimeActivity()` call to the `init` of the `NewActivityViewModel`?